components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.spec.ts

```diff
@@ -265,8 +267,8 @@ describe('ProjectsFilterDropdownComponent', () => {
       });
 
       it('does nothing', () => {
+        expect(component.resetOptions).toHaveBeenCalled();
```

Comment:
not to self - this needs to be updated, test name and function is no longer accurate